app/graphql/resolvers/create_user.rb

```diff
@@ -8,15 +8,28 @@ class Resolvers::CreateUser < GraphQL::Function
 
   type Types::UserType
 
-  def call(_obj, args, _ctx)
-    User.create!(
+  def call(_obj, args, ctx)    
+    if !admin_is_valid(ctx)
+      return GraphQL::ExecutionError.new("Invalid user token. Please log in.")
+    end
+
+    emailUser = User.find_by(email: args["email"])
```

Comment:
little awkward for a name

Reply:
also camelCase is not ruby convention

Reply:
suggestions? I've thought hard about it, and it's what I settled for.

Reply:
preexisting_user? goddamn_user? I guess email_user is fine